sqlite_test.go

```diff
@@ -28,6 +28,20 @@ func TestFileOpen(t *testing.T) {
 			npages:  4,
 			pagesz:  1024,
 		},
+		{
+			// Chrome history sqlite db
+			fname: "testdata/History",
```

Comment:
could we have a more interesting name for this file?
`chrome-history.sqlite` would work with me :)